### ISSUE TITLE
[UI] Add ways to collect all windows spawn by a particular parent 

### ISSUE THREAD

**Issue by tbkoay** [org-arl/r2c2#14]
When one connects to more than one vehicle and each spawns multiple child windows. It could get confused to know which window belongs to which vehicle. Suggest the followings

- [ ] Provide a button to collect all windows spawned by a vehicle.
- [x] Provide a close button to close all windows spawned by a vehicle.
- [x] Indicate vehicle info at the windows

Existing work, added for completeness of tracking.

Priority: HIGH

**Comment by yashdeep97** [comment]
Does the first point("Provide a button to collect all windows spawned by a vehicle.") indicate that the windows must come into focus(in front of other windows)?

**Comment by tbkoay** [comment]
Yes. That would allow us to see all the relevant windows of this particular vehicle.
